pkg/agent/agent_windows.go

```diff
@@ -150,14 +150,14 @@ func (i *Initializer) prepareOVSBridge() error {
 	if err = util.SetAdapterMACAddress(brName, &uplinkNetConfig.MAC); err != nil {
 		return err
 	}
-	// Remove existing IP addresses to avoid a candidate error of "Instance MSFT_NetIPAddress already exists" when
-	// adding it on the adapter.
-	if err = util.RemoveIPv4AddrsFromAdapter(brName); err != nil {
-		return err
-	}
 	// TODO: Configure IPv6 Address.
 	if err = util.ConfigureInterfaceAddressWithDefaultGateway(brName, uplinkNetConfig.IP, uplinkNetConfig.Gateway); err != nil {
-		return err
+		if strings.Contains(err.Error(), "Instance MSFT_NetIPAddress already exists") {
+			err = nil
+			klog.V(4).Infof("Address: %s already exists on interface %s", uplinkNetConfig.IP.String(), brName)
+		} else {
+			return err
+		}
```

Comment:
nit: dealing error first can keep the normal path at a minimal indentation.
```
if !strings.Contains(err.Error(), "Instance MSFT_NetIPAddress already exists") {
	return err
}
klog.V(4).Infof("Address: %s already exists on interface %s", uplinkNetConfig.IP.String(), brName)
err = nil
```

Reply:
Thanks Quan, Good suggestion. Done.